test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -407,6 +419,11 @@ protected bool TestCore(RunContextBase ctx, string cmdName, string dataPath, str
             return TestCoreCore(ctx, cmdName, dataPath, PathArgument.Usage.DataModel, null, ctx.ModelPath(), loaderArgs, extraArgs, toCompare);
         }
 
+        protected bool TestCore(RunContextBase ctx, string cmdName, string dataPath, string loaderArgs, string extraArgs, int digitsOfPrecision, params PathArgument[] toCompare)
```

Comment:
>digitsOfPrecision [](start = 126, length = 17)

I don't see `digitsOfPrecision` being used in the method. Am I missing something?